<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xfserialport</class>
 <widget class="QDialog" name="xfserialport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xfserialport</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>0</y>
     <width>91</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="BtnOpen">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSend">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="EditRev">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditSend">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>210</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
